angular.json

```diff
@@ -201,6 +201,7 @@
   },
   "defaultProject": "ngx-domoboard",
   "cli": {
-    "defaultCollection": "@nativescript/schematics"
+    "defaultCollection": "@nativescript/schematics",
+    "analytics": "3c4eed1d-a250-4e4e-a301-ae90e1a77caf"
```

Comment:
The line `"analytics": "3c4eed1d-a250-4e4e-a301-ae90e1a77caf"` was inheriated from the more earlier version, I agree that we can remove this line and remain this file unchanged in the current version in master repository since it does not have any ripple effects.

Reply:
Can you remove it in the PR?